machine-learning/automl/requirements.txt

```diff
@@ -0,0 +1,7 @@
+pycaret[analysis,models,tuner,parallel,test]==3.1.0
+crate[sqlalchemy]==0.34.0
+tqdm==4.66.1
+mlflow-cratedb
+plotly==5.18.0
+python-dotenv==1.0.0
+werkzeug==2.2.3
```

Comment:
When doing another iteration, maybe sort them alphabetically. Otherwise, we can also do it later.